src/collections/resources/faq/service-mesh-faq.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: "Service Mesh FAQs"
+thumbnail: ../../../assets/images/service-mesh-icons/linkerd/horizontal/color/linkerd-horizontal-color.png
```

Comment:
Using just the Linkerd image is not appropraite. You can just use the `../service-mesh.svg` for now